fiftyone/core/utils.py

```diff
@@ -2893,6 +2893,51 @@ def recommend_process_pool_workers(num_workers=None):
     return num_workers
 
 
+def get_cpu_count():
```

Comment:
@swheaton if this is a more robust/reliable than always using `multiprocessing.cpu_count()`, then we should replace all occurrences of `multiprocessing.cpu_count()` with `fou.get_cpu_count()` in the codebase!

Any instances of `multiprocessing.cpu_count()` are almost certainly trying to achieve the same semantic meaning (how many CPUs are available to me?)